### ISSUE TITLE
Urinoma

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11313]
**Preferred term label:**

**Synonyms**
Urine collection

**Definition (free text, please give PubMed ID)**
PMID: 34507797 Definition: **Urinomas** are encapsulated collections of urine around the kidney. Some speculate that the kidney rupture that gives
rise to a urinoma serves as a protective mechanism for the renal parenchyma in the setting of increased pressure due to obstruction. Others interpret the presence of a urinoma as evidence of an already compromised renal parenchyma.
(...) Associated Anomalies: Urinoma is strongly associated with underlying urinary tract obstruction, most commonly because of posterior urethral
valves or ureteropelvic junction (UPJ) obstruction

https://doi.org/10.53347/rID-21472 : **Urinomas**, or uriniferous fluid collections, are urine collections usually found in the retroperitoneum, most commonly in the perirenal space, as a consequence of renal tract leakage caused by urinary obstruction, trauma, or post-instrumentation.
(...) Urinomas are most often due to urinary obstruction secondary to:
    calculus, ureteropelvic  junction obstruction, retroperitoneal fibrosis, retroperitoneal malignancy, renal pelvis and various conditions causing bladder outlet obstruction
Urinomas also may be due to: abdominopelvic trauma, e.g. renal trauma, surgery, e.g. ureteric injury, diagnostic instrumentation, e.g. ESWL, ureteroscopy

**Parent term (use hpo.jax.org/app)**
Peritoneal mass HP:0034971

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Posterior urethral valve ORPHA:93110

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
note I moved the parent to Retroperitoneal mass

Adding new term: 
- ID: HP:6001415
- label: Urinoma
- parents: [Retroperitoneal mass (HP:6000285)](https://hpo.jax.org/app/browse/term/HP:6000285)
- pmid: PMID:34507797
- def: Encapsulated collections of urine around the kidney in the retroperitoneum, most commonly in the perirenal space, as a consequence of renal tract leakage caused by urinary obstruction, trauma, or post-instrumentation.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11313
- synonyms:
